datascience-notebook/Dockerfile

```diff
@@ -4,23 +4,29 @@ LABEL maintainer=analytics-platform-tech@digital.justice.gov.uk
 
 USER root
 
-ENV PATH=$PATH:$HOME/.local/bin
-ENV NB_UID=1001
-ENV CHOWN_HOME=no
+ENV PATH=$PATH:$HOME/.local/bin \
+    NB_UID=1001 \
+    CHOWN_HOME=no
 
 COPY ./files/apt_packages /tmp/
 
-RUN conda install boto3 black flake8 jupyterlab_code_formatter \
-    && apt-get update \
-    && apt-get install -y $(cat /tmp/apt_packages) \
+RUN apt-get update \
+    && apt-get install -y \
+    ca-certificates-java \
+    openjdk-8-jdk \
+    openssh-client \
+    software-properties-common \
+    gdal-bin \
+    libspatialindex-dev \
```

Comment:
Shouldn't we keep refactorings separate from this?

I guess it may be worth doing these as you're already here but there is always the risk that something changes, especially as these would invalidate the cache and trigger the re-run of `apt-get update` etc...

Reply:
Also, if you expand/inline the `apt_packages` file AFAICT that COPY above may not be necessary anymore? 🤷‍♂️

Reply:
This was mostly me trying to get it working, and pulling in the apt packages simplified the build process. 

The only major change I've made here was conda-->pip which decreased the build time for 5+ minutes. Which is fine for now whilst I'm building it over and over and over again! 

that aside. Yes generally want to avoid refactoring, so good call 😃